source/administration/changelog.md

```diff
@@ -4,6 +4,80 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)
+
+**Release Day: 2021-01-16**
+
+Mattermost v5.31.0 contains a low level security fix. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.org/responsible-disclosure-policy/).
+
+### Compatibility
+
+ - Support for Mattermost Server v5.25 [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.31 or later is highly recommended.
```

Comment:
```suggestion
 - Support for Mattermost Server v5.25 [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) is coming to the end of its life cycle on April 16, 2021. Upgrading to Mattermost Server v5.30 or later is highly recommended.
```

Reply:
@justinegeffen v5.31 is the new ESR, and this is the reason why we're recommending upgrading to this version.

Reply:
I'll clarify the docs.